src/main/java/com/spotify/ffwd/http/HttpClient.java

```diff
@@ -53,6 +53,16 @@
         }).retryWhen(new RetryWithDelay(retries, baseDelayMillis, maxDelayMillis));
     }
 
+    public Observable<Void> sendBatch(final com.spotify.ffwd.http.model.v2.Batch batch) {
```

Comment:
Can you also move the v1 `Batch` class as part of this? It should use the same package schema for consistency (`com.spotify.ffwd.http.model.v1`).

Reply:
Done